### PR TITLE
Favorites component

### PR DESCRIPTION
**Pull request by sperrye**
## Issue Number:
**83**

## Issue Description:
**Feature request: create a preact favorites component.**

### Summary of solution:
1. Created preact component Favorites with dummy data.
2. Imported Favorites component into routes/profile.
3. Added basic styling with colors from green-theme.png.

### Can this issue be closed?
Yes

### Should any new issues be added as a result of this solution?
**Create child component 'Favorite' to house bookmark data in favorites list.**

### CODE REVIEW

File: client/src/routes/profile/index.js [hunk 1]
```diff
@@ -41,6 +42,8 @@ export default class Profile extends Component {
 					{' '}
 					Clicked {count} times.
 				</p>
```
Comment (motosharpley) [inline client/src/routes/profile/index.js]: Let's go ahead and remove the click counter while we are here too it is just left over from the preact-cli example.

File: client/src/routes/favorites/index.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import { h, Component } from 'preact';
```
Comment (motosharpley) [inline client/src/routes/favorites/index.js]: With the way you are using this, bringing it in to the routes/profile component it should live in the components directory. Reserve the routes directory for components that are complete views made up of a collection of components like this one.

**Comment by sperrye** [comment]
I've moved the favorites component to the correct directory and removed the counter feature from Profile as requested.

**Comment by motosharpley** [comment]
Looks good Thanks for contributing!

**Pull request merged**